app/_hub/kong-inc/hmac-auth/index.md

```diff
@@ -61,7 +61,7 @@ params:
   service_id: true
   route_id: true
   consumer_id: false
-  protocols: ["http", "https"]
+  protocols: ["http", "https", "grpc", "grpcs"]
```

Comment:
Need to add a gRPC example?

Reply:
hard to calculate the content hmac without collaboration with the client itself....